demo-java/app/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="com.example.placesdemo">
 
+  <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
+  <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
```

Comment:
Duplicate of L21?

Reply:
Removed both since this permission is included by Places SDK